src/client/Container/UniqueKeyPolicy.ts

```diff
@@ -0,0 +1,7 @@
+export interface UniqueKeyPolicy {
```

Comment:
Don't you need to update ./index.ts to export this type to avoid weird import paths?

Reply:
Yep. Added.